### PR TITLE
API-1835: Bump library-go to get SSA in multiple controllers

### PR DESCRIPTION
**Pull request by bertinatto**
/assign @deads2k @p0lyn0mial

### CODE REVIEW

**Comment by bertinatto** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.18-e2e-vsphere-ovn

**Comment by bertinatto** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.18-e2e-vsphere-ovn

File: pkg/operator/replacement_starter.go [hunk 1]
```diff
@@ -282,7 +282,7 @@ func CreateOperatorStarter(ctx context.Context, authOperatorInput *authenticatio
 	ret.ControllerRunFns = append(ret.ControllerRunFns, libraryapplyconfiguration.AdaptRunFn(resourceSyncer.Run))
 	ret.ControllerRunOnceFns = append(ret.ControllerRunOnceFns, libraryapplyconfiguration.AdaptSyncFn(authOperatorInput.eventRecorder, resourceSyncer.Sync))
 
-	configOverridesController := unsupportedconfigoverridescontroller.NewUnsupportedConfigOverridesController(authOperatorInput.authenticationOperatorClient, authOperatorInput.eventRecorder)
+	configOverridesController := unsupportedconfigoverridescontroller.NewUnsupportedConfigOverridesController("openshift-auhentication", authOperatorInput.authenticationOperatorClient, authOperatorInput.eventRecorder)
```
Comment (deads2k) [inline pkg/operator/replacement_starter.go:285]: if you're rebasing anyway, fix the typo in authentication.

**Pull request merged**